### ISSUE TITLE
How to change location while in motion

### ISSUE THREAD

**Issue by edgars93** [scihant/CTPanoramaView#2]
I couldn't change location/position of motion, i.e, I'm trying that UIPanGesture when is ended, change to motion automatically, but the location must be the same as location(x,y,z) before that UIPanGesture has ended, just like Facebook.
But when is ended, the motion changes automatically but the position always resets depends of the orientation.

Sorry for my english

**Comment by scihant** [comment]
That's the intended behavior. When you enable motion control, the orientation of the camera is determined solely by the motion data according to some fixed reference point. Since the reference point is the same, keeping the device in the same orientation always results in the exact FoV of the camera.  In short, you can't mix pan and motion as in the Facebook app. The reason I implemented it that way is to keep the correspondence between the physical orientation of the device and the FoV consistent.

With all these said, I understand that not everyone would care about a fixed reference point, and prefer the Facebook way. That's why I might add that feature in a future release.

**Comment by edgars93** [comment]
I don't understand. I'm learning how to use SceneKit for 360 photos
Are you saying I need to change values of xFov and yFov?

**Comment by scihant** [comment]
No, I'm saying that this library automatically sets the FoV according to the orientation of the device, regardless of how or when you modified the FoV before. As soon as you enable motion controls, it will immediately change to FoV to where it should be according to a fixed reference point. Therefore it's not possible to use touch controls without disabling the motion controls, or changing how the library works.

**Comment by edgars93** [comment]
Could you give some advice how to make this happen, please?
